public/scripts/suggestion.js

```diff
@@ -0,0 +1,22 @@
+$( document ).ready(function() {
+ var insertedItem = [];
+ $('#git_url').editableSelect();
+ $('#git_url').keyup(() => {
+   var q = $("#git_url").val();
+   if (q.length > 3) {
+     $.get(`https://api.github.com/search/repositories?q=${q}+fork%3Atrue`,
+     (result) => {
+       if (result.items != undefined) {
+         for (var i = 0; i < result.items.length; i++) {
+           if (insertedItem.indexOf(result.items[i].clone_url) < 0) {
+             insertedItem.push(result.items[i].clone_url);
+             $('#git_url').editableSelect('add', result.items[i].clone_url);
+           }
+         }
+       }
+     })
+   } else if(q.length == 0) {
```

Comment:
yeah, but then https://github.com/fossasia/yaydoc/pull/161#issuecomment-310605604 still happens for query length 1 and 2. maybe make that if q.length == 0 into an else